code/datums/traits/_quirk.dm

```diff
@@ -65,22 +65,78 @@
 		return
 	on_process()
 
-/mob/living/proc/get_trait_string(medical) //helper string. gets a string of all the traits the mob has
+/mob/living/proc/get_trait_string(medical, category) //helper string. gets a string of all the traits the mob has
 	var/list/dat = list()
-	if(!medical)
-		for(var/V in roundstart_quirks)
-			var/datum/quirk/T = V
-			dat += T.name
-		if(!dat.len)
-			return "None"
-		return dat.Join(", ")
-	else
-		for(var/V in roundstart_quirks)
-			var/datum/quirk/T = V
-			dat += T.medical_record_text
-		if(!dat.len)
-			return "None"
-		return dat.Join("<br>")
+	switch(category)
+		if(0)
```

Comment:
magic numbers

Reply:
Essentially to fix this you need to do defines for the categories.
Something like
```
#define CAT_MINOR_DISSABLIITY 2
#define CAT_MAJOR_DISSABLIITY 3
```
Then you replace the `if(2)` for `if(CAT_MINOR_DISSABLIITY)` and it's more readable.
The exact name of the define is up to you, just follow the convention of making it all in caps, try for them to have the same beginning or end, and try to make it unique enough not to risk being duplicated in the future.